src/panels/config/automation/trigger/types/ha-automation-trigger-conversation.ts

```diff
@@ -39,6 +41,12 @@ export class HaConversationTrigger
                 iconTrailing
                 .index=${index}
                 .value=${option}
+                .validationMessage=${this.hass.localize(
+                  "ui.panel.config.automation.editor.triggers.type.conversation.no_punctuation"
+                )}
+                autoValidate
+                validateOnInitialRender
```

Comment:
By design that `validateOnInitialRender` is not set for the textfield below in the other code change block?

Reply:
Yes, that will always be empty on initial render :-)